config/application.rb

```diff
@@ -31,7 +31,7 @@ class Application < Rails::Application
     config.autoload_paths += config.eager_load_paths
 
     config.autoload_paths << Rails.root.join('lib').to_s
-
+    config.log_level = (ENV['RAILS_LOG_LEVEL'] || 'debug').downcase.to_sym
```

Comment:
If this runs before `config/environments/production.rb`, it's just going to get overwritten in that file. So I think we should remove `config.log_level` from the `production.rb` file, otherwise this seems good to me 👍